### DISCUSSION TITLE
Where to get IP and port for the Eufy Security Web socket from?

### DISCUSSION THREAD

**Discussion by mohammad-shadmehr** [fuatakgun/eufy_security#784] [Q&A]
Where to get IP and port for the Eufy Security Web socket from? I followed the installation steps and got stuck at "5- Put Eufy Security Add-on IP Address (127.0.0.1 for Supervised installation) and configured port (default 3000) and click Submit."
It keeps giving "incorrect IP/port" errors.
Do I need to install anything else prior installing the Eufy security Add on?

**Comment by Plicerin** [comment]
same

**Comment by copadewe** [comment]
same here, just tried to install but stuck at this point 5. What should that be? Where to find this? Port 3000? I have no idea if URL or Port is wrong, and missing the info, what exactly this should be.
Tried the 127.0.0.... or the url from the eufy base, or my fritz.box host # nothing

Thx
![Uploading eufy.png…]()

**Reply by mgohome** [reply]
I have the same problem, can not get further

**Reply by MrEMan** [reply]
Think I can help here. I reinstalled but forgot this crucial step.

Installing the required add-on https://github.com/bropat/hassio-eufy-security-ws is not enough. You need to configure and start it.

Go to Settings -> Add-ons
[https://your-instance.duckdns.org/hassio/dashboard](https://your-instance.duckdns.org/config/integrations)

Find eufy-security-ws complete the configuration then click start

Regards

**Reply by protomota** [reply]
Oh why oh why is this not mentioned in the documentation? I was completely stuck on this until I found your comment.

**Reply by rkbest13** [reply]
Why this important step is not mentioned anywhere on github steps. I gave up last time i tried and now found this after few months.

**Reply by couzin2000** [reply]
> Think I can help here. I reinstalled but forgot this crucial step.
> 
> Installing the required add-on https://github.com/bropat/hassio-eufy-security-ws is not enough. You need to configure and start it.
> 
> Go to Settings -> Add-ons [https://your-instance.duckdns.org/hassio/dashboard](https://your-instance.duckdns.org/config/integrations)
> 
> Find eufy-security-ws complete the configuration then click start
> 
> Regards


[MrEMan](https://github.com/MrEMan) does "Go to Settings -> Add-ons // [https://your-instance.duckdns.org/hassio/dashboard](https://your-instance.duckdns.org/config/integrations)" mean I need to sign up for DuckDNS dynamic DNS service? If so, is there no other way around this? I use No-IP.com...

**Comment by HSA1963** [comment]
I have installes te hassio-eufy-security-ws via HACS then looking in Add-ons there is no eufy security? But in devices and services I find the Eufy Securiy app now. But then asking IP adres and port. 127.0.0.0 and 3000 is not working. my HA IP adress and port 3000 or 8123 is not working. Also IP 0.0.0.0 and port 3000 is not working. Where can I find the right IP adress and Port which I need to use. Spent already a lot of time reading all related issues and nothing is working. Or there is a lack of information or I'm to stupid to untherstand the sollution.

**Comment by machadojon** [comment]
I haven't been able to confirm, but looks like that you need a eufy home base to make this integration works. Is that correct?
